package.json

```diff
@@ -11,15 +11,12 @@
     "postcompile": "node scripts/postcompile.mjs",
     "watch": "tsc --build tsconfig.esm.json --watch",
     "install-with-npm-8.5": "npm i -g npm@^8.5.0 && npm i",
-    "pretest": "tsc --build tsconfig.json",
+    "pretest": "npm run compile",
     "test": "jest --verbose",
-    "test:clean": "jest --clearCache",
     "test:watch": "jest --verbose --watchAll",
-    "test:smoke": "npm run test:smoke:prepare && npm run test:smoke:run",
-    "test:smoke:prepare": "smoke-test/prepare.sh",
-    "test:smoke:run": "smoke-test/smoke-test.sh",
-    "testonly": "npm test",
-    "test:ci": "npm run coverage -- --ci --maxWorkers=2  --reporters=default --reporters=jest-junit",
+    "pretest:smoke": "npm run compile && smoke-test/prepare.sh",
```

Comment:
eh, you might want to be able to manually run the test itself without re-running the prepare:

- just because the prepare is kinda slow
- or if you've directly edited some random file inside `smoke-test/node_modules` to see what it does

(I guess in that case you could just run `smoke-test/smoke-test.sh` manually though i think it might depend on the `npm run` environment a bit)

Reply:
Reverted. Originally my intended outcome here was to make sure `compile` ran as part of smoke prepare. But that already happens in the `prepare.sh` file so no change actually needed here in the first place.